tyk-headless/values.yaml

```diff
@@ -136,10 +136,14 @@ pump:
   enabled: false
 
   replicaCount: 1
+  # containerPort: 9090
```

Comment:
For setting/adding this containerPort, it needs to explicitly opened from Pump container. I don't think we have any port open by default. Do we have any ENV var readily available to open a port on fly ?

Reply:
Yes, this is support for pumps like Prometheus which require the pump to expose a service to scrape metrics. Or if someone wants to have multiple Prometheus pumps with different data

Reply:
Got it ! I was thinking in the lines that are exposing pump as a generic service rather than only targeting prom pump, so this looks good.